README.md

```diff
@@ -59,8 +58,7 @@ Functional examples are included in the
 | all\_folder\_admins | List of IAM-style members that will get the extended permissions across all the folders. | list | `<list>` | no |
 | folder\_admin\_roles | List of roles that will be applied to per folder owners on their respective folder. | list | `<list>` | no |
 | names | Folder names. | list | `<list>` | no |
-| parent\_id | Id of the resource under which the folder will be placed. | string | n/a | yes |
-| parent\_type | Type of the parent reosurce, defaults to organization. | string | `"organization"` | no |
+| parent | IThe resource name of the parent Folder or Organization. Must be of the form folders/folder_id or organizations/org_id | string | n/a | yes |
```

Comment:
Was this description manually added? You should use the generator to create docs.

Reply:
I has some issues with terraform-docs and v12, but figured out ho to fix, the docs have been updated with auto-generated